scripts/xtensa-build-all.sh

```diff
@@ -8,7 +8,7 @@ set -e
 # Platforms with a toolchain available in the latest Docker image and
 # built by the -a option.
 DEFAULT_PLATFORMS=(  byt cht bdw hsw tgl tgl-h apl skl kbl cnl sue icl jsl \
-                    imx8 imx8x imx8m imx8ulp rn mt8186 mt8195 )
+                    imx8 imx8x imx8m imx8ulp rn rmb mt8186 mt8195 )
```

Comment:
The docker container does not have the `xtensa-rmb-elf` toolchain yet, so this change will break `./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -a` for everyone. Add `rmb` to the next line instead:

```
SUPPORTED_PLATFORMS=( "${DEFAULT_PLATFORMS[@]}" rmb )
```

See comments here and similar discussion in #5388

Reply:
overlay and corresponding configs got up streamed and merged for rembrandt platform. So with above patch sets GCC build won't break.
Quoting the Pull requests for reference
https://github.com/thesofproject/crosstool-ng/pull/16
https://github.com/thesofproject/xtensa-overlay/pull/13

Reply:
For now `rmb` still fails in https://github.com/thesofproject/sof/runs/6759196854 submitted 6 hours ago.

Please resolve comments when they are actually resolved, not when they "will be" resolved.

BTW
```diff
--- a/.github/workflows/pull-request.yml
+++ b/.github/workflows/pull-request.yml
@@ -114,7 +114,7 @@ jobs:
         # does not block all other builds.
         platform: [imx8ulp,
                    sue jsl tgl,
-                   rn,
+                   rn rmb,
                    mt8186 mt8195,
         ]
```

Reply:
For now `rmb` still fails in https://github.com/thesofproject/sof/runs/6759196854 submitted 6 hours ago.

Please resolve comments when they are actually resolved, not when they "will be" resolved.

BTW
```diff
--- a/.github/workflows/pull-request.yml
+++ b/.github/workflows/pull-request.yml
@@ -114,7 +114,7 @@ jobs:
         # does not block all other builds.
         platform: [imx8ulp,
                    sue jsl tgl,
-                   rn,
+                   rn rmb,
                    mt8186 mt8195,
         ]
```